src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString.split(';');
+  const clearStyles = styles
```

Comment:
The variable name 'clearStyles' could be improved to reflect what it represents more accurately. A name like 'filteredStyles' or 'nonEmptyStyles' might give a better indication of its contents.